src/web/routes/application/select-address/adapters.addresses.test.js

```diff
@@ -0,0 +1,193 @@
+const test = require('tape')
+const { transformAddress } = require('./adapters')
+
+test('transformAddress() builds address A', (t) => {
```

Comment:
I know we don't normally do this, but would it be worthwhile running these examples past Karen before we merge them in, might be worthwhile to double check with her first. I'm happy with them though.

Reply:
Yes I'm happy to do that, @YetAnotherMatt what do you think (as you were discussing the format with Karen)?

Reply:
Good idea to share them with Karen, though I'm not sure we need to hold up merging the PR - we can always address any changes before the story is completed.

Reply:
It seems I replied in the wrong place for it to show up here...
Good idea to share them with Karen, though I'm not sure we need to hold up merging the PR - we can always address any changes before the story is completed.